flink-streaming-java/src/main/java/org/apache/flink/streaming/util/keys/KeySelectorUtil.java

```diff
@@ -67,7 +67,7 @@
 
 	public static <X> ArrayKeySelector<X> getSelectorForArray(int[] positions, TypeInformation<X> typeInfo) {
 		if (positions == null || positions.length == 0 || positions.length > Tuple.MAX_ARITY) {
-			throw new IllegalArgumentException("Array keys must have between 1 and " + Tuple.MAX_ARITY + " fields.");
+			throw new IllegalArgumentException("The fields of array keys should between 1 and " + Tuple.MAX_ARITY + ".");
```

Comment:
There is no verb in the sentence. Moreover how does it improve the original message?